plugins/exporter.koplugin/clip.lua

```diff
@@ -9,7 +10,7 @@ local T = require("ffi/util").template
 
 local MyClipping = {
     my_clippings = "/mnt/us/documents/My Clippings.txt",
-    history_dir = "./history",
+    history_dir = string.format("%s%s", DataStorage:getDataDir(), "/history"),
```

Comment:
You can use string.format if you want, but it'll also be just fine and imo more legible to do:
```suggestion
    history_dir = DataStorage:getDataDir() .. "/history",
```
(Not too long ago `string.format` was JIT compiled while concatenation was interpreted so back then there was a reason to prefer `string.format`.)

Reply:
Both work for me :+1: